CHANGELOG.next.asciidoc

```diff
@@ -89,6 +89,72 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- Add AWS SQS metricset. {pull}10684[10684] {issue}10053[10053]
```

Comment:
This looks like it contains a bunch of unrelated changelog entries?

Reply:
Yup, bad rebase. Fixing...

Reply:
Fixed in a7c312d.